disperser/apiserver/server_v2.go

```diff
@@ -254,20 +254,22 @@ func (s *DispersalServerV2) GetPaymentState(ctx context.Context, req *pb.GetPaym
 	currentReservationPeriod := meterer.GetReservationPeriod(now, reservationWindow)
 	binRecords, err := s.meterer.OffchainStore.GetBinRecords(ctx, req.AccountId, currentReservationPeriod)
 	if err != nil {
-		return nil, api.NewErrorNotFound("failed to get active reservation")
+		s.logger.Debug("failed to get reservation records, use placeholders", err, accountID)
 	}
 	largestCumulativePayment, err := s.meterer.OffchainStore.GetLargestCumulativePayment(ctx, req.AccountId)
 	if err != nil {
-		return nil, api.NewErrorNotFound("failed to get largest cumulative payment")
+		s.logger.Debug("failed to get largest cumulative payment, use zero value", err)
 	}
 	// on-Chain account state
 	reservation, err := s.meterer.ChainPaymentState.GetReservedPaymentByAccount(ctx, accountID)
 	if err != nil {
-		return nil, api.NewErrorNotFound("failed to get active reservation")
+		s.logger.Debug("failed to get onchain reservation, use zero values", err)
+		reservation = core.DummyReservedPayment()
```

Comment:
why don't we leave them as nil?

Reply:
The way we [build reply](https://github.com/Layr-Labs/eigenda/blob/07670726c824e8228503a2799efc7183db73dab5/disperser/apiserver/server_v2.go#L291) struct assumes that there is a valid nonnil object. Would it make more sense to set the inner fields here and use those when we construct the reply?

Reply:
Does the reply need to return zero valued object? Could we return nil instead?
Accountant handles nil reservation/ondemand payment [here](https://github.com/Layr-Labs/eigenda/pull/1014/files/07670726c824e8228503a2799efc7183db73dab5#diff-12b74ec71c12c2e17777a7dee108651d3a1cd184a32a2bb522a43c0f0baa204dR187)

Reply:
We could return nil, it is just that we will add a separate step to convert cached struct to protobuf structs outside of the reply construction. 

This seems like a cleaner behavior, so I will update to do that